change/@azure-msal-browser-adabc718-b19b-4b98-a5b7-887465c75857.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
I think this can be a patch, no api changes.

Reply:
I still think it is a significant enough feature. Is the "minor" tag only for API changes? My understanding is "major" is needed for public API and other tags can be decided based on the feature impact??

Reply:
Probably doesn't matter for this PR since we're already going to do a minor version bump due to other PRs, but we should come to a consensus on this. I would personally consider this a minor feature rather than a patch

Reply:
I would classify this as fixing broken behavior, given the expectation from customers that MSAL should more gracefully handle concurrent requests out of the box, given we don't define it as a requirement today that you have to make all acquireTokenSilent calls in serial.

Reply:
Already a queued up minor change, so we can keep this as-is, resolving.

Reply:
+1 for minor